RecoTauTag/RecoTau/interface/AntiElectronIDMVA6.h

```diff
@@ -146,6 +183,10 @@ class AntiElectronIDMVA6 {
   float* Var_NoEleMatch_wGwoGSF_Endcap_;
   float* Var_woGwGSF_Endcap_;
   float* Var_wGwGSF_Endcap_;
+  float* Var_NoEleMatch_woGwoGSF_VFEndcap_;
+  float* Var_NoEleMatch_wGwoGSF_VFEndcap_;
+  float* Var_woGwGSF_VFEndcap_;
+  float* Var_wGwGSF_VFEndcap_;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.

Reply:
It is following convention from previous version, but I will change it here and older variables of this kind.

Reply:
Thank you.
Large enough modifications are a good reason to update the non-compliant old code.